src/controls.plugin.js

```diff
@@ -0,0 +1,60 @@
+function wikiaJWPlayerRecommendedControlsPlugin(player, config, div) {
```

Comment:
I'd name this function `wikiaJWPlayerSmallPlayerControls` to be consistent with the name of the plugin. Same thing with the filename - `small-player-controls.plugin.js`?